src/simply/simply_stage.c

```diff
@@ -200,6 +216,12 @@ static void circle_element_draw(GContext *ctx, SimplyStage *self, SimplyElementC
   }
 }
 
+static void circlearc_element_draw(GContext *ctx, SimplyStage *self, SimplyElementCircleArc *element) {
+  GRect frame = grect_inset(element->frame, GEdgeInsets(30));
+  graphics_context_set_fill_color(ctx, element->background_color);
+  graphics_fill_radial(ctx, frame, GOvalScaleModeFitCircle, element->radius, DEG_TO_TRIGANGLE(element->anglestart), DEG_TO_TRIGANGLE(element->angleend));
```

Comment:
Newer code is restricted to a 100 long line limit. This can be:
```c
  graphics_fill_radial(ctx, frame, GOvalScaleModeFitCircle, element->radius,
                       DEG_TO_TRIGANGLE(element->angle_start),
                       DEG_TO_TRIGANGLE(element->angle_end));
```